README.md

```diff
@@ -8,6 +8,62 @@ The main idea is to have very simple but flexible development workflow and deplo
 4. SCSS support is also included (```watch``` script automatically rebuilds your CSS from SCSS after any changes).
 5. Multiple languages are supported, see [strings.json](./strings.json).
 
+## How it works
+Let's pick "team" page for example to see how it works. Pages of your site are defined in [config.php](https://github.com/deathbaba/landing-php/blob/master/config.php) `$PAGES` array. There is an entry `team.php` in `$PAGES` that defines "team" page:
+
+```
+'team.php' => [
+  'link' => 'team',
+  'menu' => 'menuTeamPage',
+  'title' => 'titleTeamPage',
+  'description' => 'metaDescriptionTeamPage',
+  'keywords' => 'metaKeywordsTeamPage'
+],
+```
+Such configuration indicates that "team" page will have url `yoursite.com/team`, menu `menuTeamPage`, title `titleTeamPage` and also description and keywords. `menuTeamPage`, `titleTeamPage` and other properties are defined in [translations/team.json](https://github.com/deathbaba/landing-php/blob/master/translations/team.json).
+The "team" page itelf is located at [www/team.php](https://github.com/deathbaba/landing-php/blob/master/www/team.php) and it first part contains all the data that page displays:
+
+```
+[
+  'img' => 'img/team/Igor_Davydov.jpg',
+  'name' => T('Igor Davydov'),
+  'title' => T('idavydovTitle'),
+  'description' => T('idavydovDescription')
+]
+```
+This entry describes team member with image path, name, title and description. Last three properties' content is defined also in [translations/team.json](https://github.com/deathbaba/landing-php/blob/master/translations/team.json). Let's take a closer look at these properties:
```

Comment:
I believe that should be described by person who have more knowledge about translation section